static/docs/commands-reference/get.md

```diff
@@ -37,6 +37,13 @@ created in the current working directory, with its original file name.
 
 ## Options
 
+- `-o`, `--out` - specify a path (directory and file name) to the desired
+  location to place the imported data in. The default value (when this option
+  isn't used) is the current working directory (`.`) and original file name.
+
+- `--rev` - specific Git revision of the DVC repository to import the data from.
+  `HEAD` by default.
```

Comment:
can we provide a link, or mention that it's usually a master's branch last commit? cc @efiop is HEAD technically correct here? just to double check

Reply:
Linked and clarified in c38e6ce.
Good question for @efiop though, I moved it to https://github.com/iterative/dvc.org/pull/601#pullrequestreview-282331391.
p.s. I just copied this from the existing `dvc import` ref.